src/components/sale/SalesLeadTable.tsx

```diff
@@ -310,6 +330,22 @@ const SalesLeadTable: React.VFC<{ sales: SalesProps; leads: LeadProps[]; onRefet
 
   return (
     <StyledAdminCard>
+      {selectedMember && (
+        <MemberPropertyModal
+          key={selectedMember.id}
+          visible={propertyModalVisible}
+          onCancel={() => setPropertyModalVisible(false)}
+          member={selectedMember}
+          sales={{
+            id: sales.id,
+            name: sales.name,
+            email: sales.email,
+          }}
+          onRefetch={() => {
+            setPropertyModalVisible(false)
+          }}
```

Comment:
the same as props `onCancel` 

`onClose` may be a better naming

Reply:
good idea